siriuspy/siriuspy/currinfo/filling_pattern.py

```diff
@@ -17,14 +17,14 @@ class DEVICES:
         SI = ('10.128.150.77', 5035)
         ALL = (SI, )
 
-    def __init__(self, devname):
+    def __init__(self, device):
```

Comment:
I think it is better to make this an optional argument.

Reply:
you think so? the way I see the class, despite its name, is quite general. it could be used to get any keysight osc. waveform, right, as long as the IP, PORT tuple is passed, right ? you suggest using DEVICES.SI as default?

Reply:
Yes, I suggest using `DEVICES.SI` as default since it greatly simplifies the creation of instances. 
Apart from that, I don't think this class is general, because the methods defined here for interaction with the Oscilloscopes are particular for this application. I understand what you mean, though, considering how easy it would be to use this code to make other classes.

Reply:
Yes, what I had in mind what eventually create a general class for Keysight oscilloscopes, without any default `device` argument, and rewrite `MeasFittingPattern` as a specialized implementation of this class. 
Ok, I have added DEVICES.SI as default device for the current `MeasFittingPattern` class.